tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>write descriptions about what as_tuple means, see similar methods in pytorch</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>